### PR TITLE
Updated the code by reducing the size of comments for better readability

### PR DESCRIPTION
**Pull request by Pranathi-star**
## Change Summary

Provide summary of changes with issue number if any.

## Checklist

**If you haven't fulfilled the below requirements, your PR won't be reviewed.**

### General 

- [x] Make sure you've read [CONTRIBUTING.md](https://github.com/wingkwong/leetcode-the-hard-way/blob/main/CONTRIBUTING.md) 
- [x] Make sure you've followed the template format
- [x] Make sure you've started the app locally and verified all the content and all links (if applicable) must be accessible correctly
- [x] Make sure you've applied LaTex for variables, formulas and time / space complexity instead of using backticks
- [x] Make sure you've formatted your code well (Code Block is rendered using `<Tabs/>`)
      Below is a C++ example. You can include multiple langs using `<TabItem/>` if necessary
      
````
<Tabs>

<TabItem value="cpp" label="C++">
<SolutionAuthor name="@YOUR_ALIAS"/>

```cpp
// Your code goes here
// Prefer K&R Coding style
```

</TabItem>

</Tabs>
````
- [x] Make sure you've explained your code well (write comments above each line - not inline)
- [x] Make sure you've written your explanation well and it is easy to understand for beginners
- [x] Make sure you've included Time & Space Complexity Analysis
- [x] Make sure you've included a meaningful approach name for your solution. Don't leave it blank. e.g. `## Approach 1: Two Pointers`.

### For Tutorial Articles

- [ ] Make sure you've explained your topic using 1 - 3 LC problems rather than using external problems. Solutions must be taken with explanations as well.
- [ ] Make sure you've provided suggested problems at the end using the given format. See [here](https://raw.githubusercontent.com/wingkwong/leetcode-the-hard-way/main/tutorials/math/number-theory/binary-exponentiation.md) as an example. If the target solution is not available, leave `solutionLink` blank.
- [ ] Make sure you've given credits / references if you use external resources. For an image, give credit under it. Otherwise, add a new section called References at the end (after Suggested Problems).

### CODE REVIEW

**Comment by wingkwong** [comment]
You are creating 2 PRs (another one: https://github.com/wingkwong/leetcode-the-hard-way/pull/206) with 2 same files which is quite confusing which is which.

**Comment by Pranathi-star** [comment]
I am sorry for the inconvenience. This PR contains the updated code for both solutions. Please ignore the previous one. I will close that.

File: solutions/0200-0299/0289-game-of-life-medium.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+description: 'Author: @Pranathi-star | https://leetcode.com/problems/game-of-life/'
+tags: [2-D Array, Iteration]
+---
+
+# 0289 - Game of Life
```
Comment (wingkwong) [inline solutions/0200-0299/0289-game-of-life-medium.md:6]: `# 0289 - Game of Life (Medium)`

File: solutions/0200-0299/0289-game-of-life-medium.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+<TabItem value="py" label="Python">
+<SolutionAuthor name="@pranathi_kodich"/>
+
+```py
+# Time Complexity: O(R * C). 
+# Each cell in the matrix is visited atmost once
+# Space Complexity: O(1). 
+# Constant space required for storing directions.
+
+class Solution:
```
Comment (wingkwong) [inline solutions/0200-0299/0289-game-of-life-medium.md:54]: Format your code well please. The indentation looks awful in code. Even tho it looks ok in UI.

File: solutions/0900-0999/0994-rotting-oranges-medium.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
+---
+description: 'Author: @Pranathi-star | https://leetcode.com/problems/rotting-oranges/'
+tags: [2-D Array, Breadth First Search]
+---
+
+# 0994 - Rotting Oranges
```
Comment (wingkwong) [inline solutions/0900-0999/0994-rotting-oranges-medium.md:6]: `# 0994 - Rotting Oranges (Medium)`

**Review by wingkwong — CHANGES_REQUESTED** [review]
Hope it's the final round.

**Comment by wingkwong** [comment]
any update?

**Comment by wingkwong** [comment]
Closing due to inactivity.

**Pull request closed without merging**